pkg/controller/projectreference/projectreference_adapter.go

```diff
@@ -635,13 +654,13 @@ func (r *ReferenceAdapter) AddOrUpdateBindings(serviceAccountEmail string) (Addo
 }
 
 // SetIAMPolicy attempts to update policy if the policy needs to be modified
-func (r *ReferenceAdapter) SetIAMPolicy(serviceAccountEmail string) error {
+func (r *ReferenceAdapter) SetIAMPolicy(serviceAccountEmail string, policies []string, group bool) error {
```

Comment:
I'd prefer to not use a bool as type for group, this makes function calls hard to read. I recommend using a type for it.

Reply:
WDYT @rafael-azevedo @sam-nguyen7 ?

Reply:
would it be a high effort to get this done here? I just fear code we add with the intention to change "later" because often enough later means never unfortunately.

Reply:
Makes sense! But tbh I cannot figure out that API and it is not a high priority. So I will change bool to a type first.

Reply:
Updated, sorry for the delay. Please take a look!